src/App.css

```diff
@@ -1,38 +1,72 @@
-.App {
+.container {
+  background-color: #E5E5E5;
+  height: 90%;
+  width: 96%;
+  margin: auto;
+  text-align: justify;
+}
+.bedTab {
```

Comment:
Once you start to feel more comfortable with your component design/composition, we can perhaps split the main App.css file into separate files that are referenced/imported directly by the specific component...

Reply:
Sounds great.